modules/nw-multus-ipam-object.adoc

```diff
@@ -310,4 +310,47 @@ pod/whereabouts-reconciler-p4sxw 1/1 Running 0 6s
 pod/whereabouts-reconciler-rvfdv 1/1 Running 0 6s
 pod/whereabouts-reconciler-svzw9 1/1 Running 0 6s
 daemonset.apps/whereabouts-reconciler 6 6 6 6 6 kubernetes.io/os=linux 6s
-----
\ No newline at end of file
+----
+
+[id="nw-multus-creating-whereabouts-dualstack-config_{context}"]
+== Creating a DualStack config
+
+DualStack can be configured with `ipRanges`.
+
+.Procedure
+
+. Set type to whereabouts
+. Use `ipRanges` to allocate IP addresses.
+
+For example, defining the range as `192.168.10.1/24` excludes the first network address and the last broadcast address.
+
+
+[source,yaml]
+----
+cniVersion: operator.openshift.io/v1
+kind: Network
+metadata:
+  name: cluster
+spec:
+  additionalNetworks:
+  - name: whereabouts-shim
+    namespace: default
+    rawCNIConfig: |-
+      {
+       "name": "whereabouts-shim",
```

Comment:
While the name is arbitrary, let's use another name here for purposes of the example, and since the shim is kind of a specific thing, maybe:

```
"name": "whereabouts-dual-stack",
```

Reply:
Thank you!